dotCMS/src/main/java/com/dotmarketing/db/DotRunnableThread.java

```diff
@@ -0,0 +1,65 @@
+package com.dotmarketing.db;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.dotmarketing.business.DotStateException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.util.Config;
+
+public class DotRunnableThread extends Thread {
+
+    final List<DotRunnable> listeners;
+
+    public DotRunnableThread(List<DotRunnable> listeners) {
+
+        this.listeners = listeners;
+    }
+
+    @Override
+    public void run() {
+        try {
+            LocalTransaction.wrap(this::_internalRunner);
+        } catch (Exception dde) {
+            throw new DotStateException(dde);
+        }
+    }
+
+    private void _internalRunner() {
+        final Set<String> reindexInodes = new HashSet<String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)